### ISSUE TITLE
NameError: global name 'behavior_tree_leaves' is not defined

### ISSUE THREAD

**Issue by jay3ss** [miccol/ROS-Behavior-Tree#8]
With the following command `rosrun behavior_tree_leaves condition_example.py` I get the following error:

```
Traceback (most recent call last):
  File "/home/user/catkin_ws/src/behavior_tree/behavior_tree_leaves/example_nodes/python/condition_example.py", line 50, in <module>
    BTAction(rospy.get_name())
  File "/home/user/catkin_ws/src/behavior_tree/behavior_tree_leaves/example_nodes/python/condition_example.py", line 20, in __init__
    self._as = actionlib.SimpleActionServer(self._action_name, behavior_tree_leaves.msg.BTAction, execute_cb=self.execute_cb, auto_start = False)
NameError: global name 'behavior_tree_leaves' is not defined
```

The issue, I believe, is in [line #20](https://github.com/miccol/ROS-Behavior-Tree/blob/master/behavior_tree_leaves/example_nodes/python/condition_example.py#L20) where `behavior_tree_leaves.msg.BTAction` is listed as an argument to the `SimpleActionServer` constructor. Should this be `behavior_tree_core.msg.BTAction` as in the [action example](https://github.com/miccol/ROS-Behavior-Tree/blob/master/behavior_tree_leaves/example_nodes/python/action_example.py#L20)? When I applied this change the node did indeed run. 

Btw, I did notice a similar [issue](https://github.com/miccol/ROS-Behavior-Tree/issues/4#issuecomment-244015081) with the C++ implementation (which seems to be resolved).

**Comment by miccol** [comment]
Thanks, I will fix that in the version 1.3 (available soon, the beta is available at the branch "version1.3")

**Comment by miccol** [comment]
Fixed. Thanks. Check out the new version.
if you like the library, please star it. Thank you.

Michele
